config/smarty.config.inc.php

```diff
@@ -118,6 +118,7 @@ function smartyEscape($string, $esc_type = 'html', $char_set = null, $double_enc
 smartyRegisterFunction($smarty, 'modifier', 'str_replace', 'str_replace');
 smartyRegisterFunction($smarty, 'modifier', 'stripslashes', 'stripslashes');
 smartyRegisterFunction($smarty, 'modifier', 'strtolower', 'strtolower');
+smartyRegisterFunction($smarty, 'modifier', 'strstr', 'strstr');
```

Comment:
```suggestion
smartyRegisterFunction($smarty, 'modifier', 'strstr', 'strstr');
smartyRegisterFunction($smarty, 'modifier', 'strtolower', 'strtolower');
```
Let's keep it in order this time to avoid further duplications ;)

Reply:
HA :D I swear to god I had you in mind :D I said "Sharak is working on making it alphjabetical, let's not mess it up".

I need to buy glasses :D